challenges/prototypes.js

```diff
@@ -5,29 +5,63 @@
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
+function  CuboidMaker(props) {
+this.length = props.length;
+this.width = props.width;
+this.height = props.height
+}
+
 
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
   
   Formula for cuboid volume: length * width * height
 */
-
+function  CuboidMaker(props) {
+  this.length = props.length;
+  this.width = props.width;
+  this.height = props.height
+  this.volume = function () {
+    return (props.length * props.width * props.height);
+  };
+}
 
 /* == Step 3: Surface Area Method ==
   Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height. 
 
   Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
 */
+function  CuboidMaker(props) {
+  this.length = props.length;
+  this.width = props.width;
+  this.height = props.height
+  this.volume = function () {
+    return (props.length * props.width * props.height);
+  };
+  this.surfaceArea = function () {
+    return 2 * (props.length * props.width + props.length * props.height + props.width * props.height);
+  };
+}
 
 
 /* == Step 4: Create a new object that uses CuboidMaker ==
   Create a cuboid object that uses the new keyword to use our CuboidMaker constructor
   Add properties and values of length: 4, width: 5, and height: 5 to cuboid.   
 */
 
+const cuboid = new CuboidMaker({
+  length: 4,
+  width: 5,
+  height: 5
+})
+
+cuboid.prototype = Object.create(CuboidMaker.prototype);
```

Comment:
What do you think line 59 is doing?

Reply:
line 58 is not necessary.  What you're doing is setting cuboid's prototype to CuboidMaker's prototype, but cuboid is created with the CuboidMaker, therefore, it already inherits the CuboidMaker's prototype.